answers/os.md

```diff
@@ -84,7 +84,18 @@ Child process cannot change variables of its parent.
 
 ## Concurrency vs Parallels? (in case single CPU core and multiple CPU cores)
 
-To be defined.
+Concurrency: executing multiple tasks at the same time but not necessarily simultaneously.
```

Comment:
Based on [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/simultaneously), simultaneously = happening or being done at exactly the same time
![image](https://user-images.githubusercontent.com/7721631/219577642-90516508-d22b-45a5-bcef-22a14136fd55.png)

So this sentence:
> executing multiple tasks at the same time but not necessarily simultaneously.

is kinda confusing.